### DISCUSSION TITLE
How form customizer app register with the SharePoint site content type?

### DISCUSSION THREAD

**Discussion by SudhirMuvva** [pnp/powershell#3282] [Q&A]
How form customizer app register with the SharePoint site content type?

I have site collection admin permission as well as Global admin. But not have access on Tenant level.

I have an Issue with PowerShell while running below line of code.

Script:
 $siteURL="https://mycompany.sharepoint.com/sites/DevSite"
Connect-PnPOnline -Url $siteURL -Interactive

Error :
Connect-PnPOnline : A parameter cannot be found that matches parameter name 'Interactive'.

I am using below powershell and sp pnp module version
PSVersion                                     : 5.1.19041.3031
SharePointPnPPowerShellOnline : 3.29.2101.0

Let me know what the issue is and why this line gives error.

**Comment by AndersRask** [comment]
You are using the legacy powershell module that is no longer maintained

This repo is for pnp.Powershell

Either upgrade to the new module by removing the old module and installing pnp.powershell or use the -useweblogin for MFA login

**Reply by SudhirMuvva** [reply]
Can you please provide the script how to attach content type to site.

Can you up[date the below script file and provide the new one
[script.txt](https://github.com/pnp/powershell/files/12092709/script.txt)

**Reply by AndersRask** [reply]
Sure. My current rate for consultency work is $250 per hour. How do you want me to bill you? :)

**Reply by AndersRask** [reply]
Joke aside, if you have a question, post that in the discussions list and maybe someone will help you out.
What you are asking for can be found with a quick google search by the way.
[how to attach content type to site](https://www.google.com/search?q=how+to+attach+content+type+to+site+pnp.powershell&ei=Krm3ZMa7F-WNxc8PifCZwAk&ved=0ahUKEwjG-JupxpqAAxXlRvEDHQl4BpgQ4dUDCA8&uact=5&oq=how+to+attach+content+type+to+site+pnp.powershell&gs_lp=Egxnd3Mtd2l6LXNlcnAiMWhvdyB0byBhdHRhY2ggY29udGVudCB0eXBlIHRvIHNpdGUgcG5wLnBvd2Vyc2hlbGwyBxAhGKABGAoyBxAhGKABGApIxiBQrwRYmCBwAXgBkAEAmAG_AaABjgmqAQQxMi4yuAEDyAEA-AEBwgIKEAAYRxjWBBiwA8ICBRAhGKABwgIIECEYFhgeGB3iAwQYACBBiAYBkAYI&sclient=gws-wiz-serp)
